File: 프로그래머스/1/86051. 없는 숫자 더하기/없는 숫자 더하기.py
def solution(numbers):
    answer2 = 0
    answer = [0 for i in range(10)]
    for i in numbers:
        answer[int(i)] = 1
    for i in range(10):
        if answer[i] == 0: answer2 += int(i)
    
    return answer2

File: 프로그래머스/1/86051. 없는 숫자 더하기/없는 숫자 더하기.cpp
#include <string>
#include <vector>
#include <bits/stdc++.h>

using namespace std;

unordered_map<int, int> m;

int solution(vector<int> numbers) {
    for(int i=1; i<=9; i++) {
        m[i] = 1;
    }
    
    for(auto t : numbers) {
        m[t] = 0;
    }
    
    int sum = 0;
    for(auto iter=m.begin(); iter != m.end(); iter++) {
        if(iter->second == 1) sum += iter->first;
    }
    return sum;
}